src/resources/PlatformResources.ts

```diff
@@ -51,6 +53,7 @@ class PlatformResources {
     source: Sources;
     field: Field;
     saml: Saml;
+    search: Search;
```

Comment:
Missing a unit test for this

Reply:
> Missing a unit test for this

Thx !! here it is:

src/resources/tests/PlatformResources.spec.ts